app/tag-management/annotation/annotation.component.ts

```diff
@@ -1,111 +1,189 @@
-import { Component, OnInit, OnDestroy } from '@angular/core';
+import { Component, OnInit, OnDestroy, ViewChild, ElementRef, HostListener } from '@angular/core';
+import { DomSanitizer, SafeHtml } from '@angular/platform-browser';
 import { ToasterService } from 'angular2-toaster';
 import { TranslateService } from 'ng2-translate';
 
+import { AnnotationTag } from './annotation-tag.model';
+import { CanvasComponent } from './canvas/canvas.component';
 import { Tag } from '../tag.model';
 import { TagService } from '../tag.service';
 
 @Component({
   moduleId: module.id,
   selector: 'hip-annotation',
-  templateUrl: 'annotation.component.html'
+  templateUrl: 'annotation.component.html',
+  styles: [`
+    .center {
+      text-align: center;
+      margin-bottom: 10px;
+    }
+  `]
 })
 export class AnnotationComponent implements OnInit, OnDestroy {
-  mainMenu = new Array<{ label: string, entries: Tag[] }>();
+
+  @ViewChild('content') content: ElementRef;
+  @ViewChild(CanvasComponent) canvas: CanvasComponent;
+
+  @HostListener('window:resize', ['$event']) onResize() {
+    this.canvasHeight = this.content.nativeElement.parentElement.offsetHeight;
+    this.canvasWidth = this.content.nativeElement.parentElement.offsetWidth;
+    for(let tag of this.tagsInDocument) {
+      tag.redrawConnection(this.canvas)
+    }
+  }
+
+  mode = 'annotate';
```

Comment:
__[tslint]__
* Declaration of public instance member variable not allowed to appear after declaration of public instance member function

<!-- Comment by [SideCI](https://sideci.com) -->